crates/api/src/publish.rs

```diff
@@ -59,6 +99,13 @@ pub trait Publish: 'static + Send + Sync + std::fmt::Debug {
         op_ids: Vec<OpId>,
         target: Url,
     ) -> BoxFut<'_, K2Result<()>>;
+
+    /// Add agent info to be published to a peer.
+    fn publish_agent(
```

Comment:
Does this need to be on the public interface? The host should publish ops but I think only K2 needs to be able to publish agent infos

Reply:
Well, this "public" interface is the module interface, so I think the space module will need this here in order to publish the agent infos when they are updated.

Reply:
Fair enough yes, okay. Resolving this